### PR TITLE
#60 Feat: Add output message for result bucket

### PR DESCRIPTION
**Pull request by nancy-gomez**

### CODE REVIEW

**Comment by nancy-gomez** [comment]
I signed it on behalf of myself. I really just need this change in order to download some stuff from the bucket the results get stored in :c

File: lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb [hunk 1]
```diff
@@ -50,10 +50,11 @@ def self.run(params)
           upload_spinner.success("Done")
         end
 
-        UI.message("Submitting job(s) to Firebase Test Lab")
         result_storage = (params[:result_storage] ||
           "gs://#{ftl_service.get_default_bucket(gcp_project)}/#{gcs_workfolder}")
+        UI.message("Test Results bucket: #{result_storage}")
```
Comment (powerivq) [inline lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb]: Wouldn't it make more sense to display this after the run completes?

Comment (nancy-gomez) [reply lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb]: I was trying to follow the order the gcloud firebase command displays info:

```
Have questions, feedback, or issues? Get support by emailing:
  ftl-ios-feedback@google.com

Uploading [file.zip] to Firebase Test Lab...
Raw results will be stored in your GCS bucket at [https://link.to.bucket]

Test [matrix-meny191ry9bxa] has been created in the Google Cloud.
Firebase Test Lab will execute your xctest test on 1 device(s).
Creating individual test executions...
```
But I saw that the first output for this plugin is the uploading to GCS, so wanted to display the link right after that log:

```
[✔] Uploading the app to GCS... Done
[16:22:13]: Test Results bucket: gs://link.to.bucket
[16:22:13]: Submitting job(s) to Firebase Test Lab
[16:22:13]: Matrix ID for this submission: matrix-3hid33tqj2asy
[✔] Starting tests...
```

But we can add it right after the 'Submitting job(s)' line instead perhaps?

Comment (powerivq) [reply lib/fastlane/plugin/firebase_test_lab/actions/firebase_test_lab_ios_xctest.rb]: Gotcha, let's do that then. Ping me after you make the change.

**Pull request merged**